clone/README.md

```diff
@@ -0,0 +1,41 @@
+# Log Cloner
+
+A log client that copies a CT log to a local database for processing.
+The tool downloads batches of leaves in parallel, but always writes them to the local database in sequence.
+This ensures there are no missing ranges, which keeps state tracking easier.
+
+One important implementation feature is that the download tools will exponentially back off if there are errors
+communicating with the log, which prevents the client from performing a DoS on any log it is downloading.
```

Comment:
nit: `, which` scans a bit ambiguously (is it the errors or the back off which ...), maybe could use `[;,-] this`?

Reply:
Done.